backend/locator/api.py

```diff
@@ -30,54 +116,110 @@ def parse_date(string):
 
 @API.errorhandler(APIError)
 def handle_invalid_usage(exc):
-    response = flask.jsonify({
-        "status": exc.status_code,
-        "message": exc.message,
-    })
-    response.status_code = exc.status_code
-    return response
+    return jsonify({"message": exc.message}, exc.status_code)
 
 
+#-------------------------------------------------------------------------------
+# Events
+
 @API.route("/events", methods=["GET"])
 def get_events():
-    return flask.jsonify({
-        "event_id": None,
-    })
+    cfg = flask.current_app.config
+    
+    # Start with all undeleted events.
+    query = SESSION.query(Event).filter_by(deleted=False)
```

Comment:
may be by default we should filter on start_date > today

Reply:
That's sort of a UI concern, isn't it?
Also, just being pedantic, "today" is time zone dependent so not the same for all clients.

Reply:
that might be too much data to return by default, but that's out of scope of the current project.